content/agent/basic_agent_usage/windows.md

```diff
@@ -19,15 +19,36 @@ further_reading:
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
+This page outlines the basic functionality of the Windows Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
 
 ### Starting and Stopping the Agent
```

Comment:
My read of this is that the header and the content don't match up.
`Starting and Stopping` the agent seems like a run time thing (i.e. after it's installed, how do you start it and stop it).  And, the first line says `the execution of the agent is controlled by a windows service`, which is also run time.  But all of the items after that are about how to install.